openfl/component/aggregator/aggregator.py

```diff
@@ -90,6 +87,10 @@ def __init__(self,
         self.write_logs = write_logs
         self.log_metric_callback = log_metric_callback
 
+        self.straggler_handling_policy = (
+            straggler_handling_policy or CutoffTimeBasedStragglerHandling()
```

Comment:
I suggest to explicitly set the `minimum_reporting` here to 50% of `len(authorized_cols)`, to cover federations with more than 2 collaborators.

Reply:
Answer: Default parameter of CutoffTimeBasedStragglerHandling() ensure that the policy is disabled as cutoff time is set to infinite. Setting minimum_reporting to another value shall have no effect. Further minimum_reporting is an integer and currently set to 1 (to handle federated experiment with a single collaborator).

Reply:
I understand, thanks for the clarification!